tests/nominal/test_MeanResponseTransformer.py

```diff
@@ -169,59 +160,22 @@ def test_output2(self):
 
 
 class TestFit:
-    def test_super_fit_called(self, mocker):
```

Comment:
please add this one back in too (again, implicit testing of error handling done in the parent class method)